net-p2p/mldonkey/mldonkey-3.1.7.ebuild

```diff
@@ -0,0 +1,184 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools eutils flag-o-matic toolchain-funcs
+
+IUSE="bittorrent doc fasttrack gd gnutella gtk guionly magic +ocamlopt upnp"
+
+DESCRIPTION="Multi-network P2P application written in Ocaml, with Gtk, web & telnet interface"
+HOMEPAGE="http://mldonkey.sourceforge.net/ https://github.com/ygrek/mldonkey"
+SRC_URI="https://github.com/ygrek/mldonkey/releases/download/release-${PV//./-}-2/${P}-2.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+
+RDEPEND="dev-lang/perl
+	dev-ml/camlp4:=
+	gd? ( media-libs/gd[truetype] )
+	gtk? (
+		gnome-base/librsvg
+		dev-ml/lablgtk[svg]
+	)
+	guionly? (
+		gnome-base/librsvg
+		dev-ml/lablgtk
+	)
+	magic? ( sys-apps/file )
+	upnp? (
+		net-libs/libnatpmp
+		net-libs/miniupnpc:=
+	)
+"
+
+DEPEND="${RDEPEND}
+	acct-user/p2p
```

Comment:
not depending on group too?

Reply:
Moved to `RDEPEND`.

About the dependency on `acct-group/p2p`, doesn't it make more sense to make it a dependency of `acct-user/p2p` as the group package is not directly required by `net-p2p/mldonkey` but by `acct-user/p2p` instead to create the group the user will belong to?
And that dependency is already automatically added by `acct-user` eclass when adding `ACCT_USER_GROUPS=( p2p )`:
```
# emerge -pvc acct-group/p2p

Calculating dependencies... done!
  acct-group/p2p-0 pulled in by:
    acct-user/p2p-0 requires acct-group/p2p
```

Reply:
OK.